src/braket/circuits/circuit.py

```diff
@@ -436,26 +438,31 @@ def _check_if_qubit_measured(
                 qubit mappings to apply to the `instruction.target`. Key is the qubit in
                 `instruction.target` and the value is what the key will be changed to.
                 Default = `None`.
+            target_qubits (QubitSet | None): the target qubits for a Noise `instruction`.
+                Default = `None`.
```

Comment:
Why do we need this if `target` is already a `QubitSetInput`, which includes `QubitSet`?

Reply:
Good point. I removed `target_qubits` and adjusted the check for `target` to account for both types.